tasklist/client/src/modules/components/BPMNDiagram/index.test.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {render, screen} from 'modules/testing-library';
+import {BPMNDiagram} from './index';
+
+describe.skip('<BPMNDiagram />', () => {
```

Comment:
Why is this skipped?

Reply:
```
unhandled error in event listener TypeError: activeLayer.getBBox is not a function
    at Canvas.viewbox (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/Canvas.js:1152:43)
    at /Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/features/overlays/Overlays.js:672:48
    at invokeFunction (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/EventBus.js:609:13)
    at EventBus._invokeListener (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/EventBus.js:452:19)
    at EventBus._invokeListeners (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/EventBus.js:426:24)
    at EventBus.fire (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/EventBus.js:374:24)
    at Canvas._setRoot (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/Canvas.js:837:18)
    at Canvas.setRootElement (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/Canvas.js:778:8)
    at Canvas.getRootElement (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/core/Canvas.js:673:15)
    at Selection.select (/Users/douglas.bouttell/Code/zeebe/tasklist/client/node_modules/diagram-js/lib/features/selection/Selection.js:97:28)
```

spent a few hours trying to figure this out and couldn't

Reply:
Let's delete this test then, it's probably something wrong with JSDOM. We can add a test with Playwright later.